.gitignore

```diff
@@ -1 +1,3 @@
 /tools/**/vendor
+meta/.DS_Store
```

Comment:
Please do not change this file. This should be in your local "global gitignore", not in each project